main.js

```diff
@@ -193,3 +197,57 @@ function showComplete(){
   startTimerButton.innerText = 'COMPLETE!';
   showElement(logActivityButton);
 }
+
+function logActivity() {
+  addToLoggedActivities();
+  renderCard();
+  hideElement(timerDisplay);
+  showElement(newActivitySection);
+}
+
+function addToLoggedActivities() {
+  if(!loggedActivities.includes(currentActivity)) {
+    currentActivity.markComplete();
+    loggedActivities.push(currentActivity);
+  }
+}
+
+function renderCard() {
+  cardsContainer.innerHTML = '';
+
+  for (var i = 0; i < loggedActivities.length; i ++) {
+    cardsContainer.innerHTML += `
+    <section class="card-flexbox">
+      <div class="log-activity-card">
```

Comment:
We might need to edit this to get the flexbox to work properly. I think it's really clever using interpolation to get to the css color property.